docs/integrations/saas-cloud/workday.md

```diff
@@ -347,7 +342,7 @@ The Workday specific configuration is shown in below two sections:
 * `Workday_Report_Config`. Remove this section to stop collecting SignOn logs from the custom report.
 * `Workday_API_Config`. Remove this section to stop collecting audit logs via REST API.
 
-You can view the entire configuration with default settings [here](https://github.com/SumoLogic/sumologic-workday/blob/master/sumoworkdaycollector/sumoworkdaycollector.yaml). For SAM deployments you can update `sumoworkdaycollector.yaml` file in the AWS Lambda console editor.
+You can view the entire configuration with default settings [here](https://github.com/SumoLogic/sumologic-workday/blob/hpal_workday_collection/sumoworkdaycollector/sumoworkdaycollector.yaml). For SAM deployments, you can update `sumoworkdaycollector.yaml` file in the AWS Lambda console editor.
```

Comment:
Hi @himanshu219 , can you confirm if this link is ok to publish, since it contains your username? The original yaml link didn't work.

Reply:
actually we are no longer maintaining this script based and SAM application based collection let me check with @yleiferman what's the plan on migrating the c2c source for fedramp environment.

Reply:
Thank you!